### DISCUSSION TITLE
Alternative PCB board manufacturer

### DISCUSSION THREAD

**Discussion by tango38317** [nliaudat/esp32_8ch_motor_shield#22] [Buy or Sell]
Hello nliaudat,
i tried to search a european company to create this board and found following company:
https://aisler.net
i also tried to upload the Gerber files to their webpage to see costs and if they can produce it.
but they give feedback there is an error (for them) on the layout:

Your board outline seems to intersect itself in at least one point. We sadly cannot process this. Common causes are overlapping segments or double/shifted contours. Please check your outline and re-upload your design.

do you have an idea what is happening, or is it only possible to order it at JLPCB?
br
Chris

**Comment by nliaudat** [comment]
I mean they cannot process the square holes (antenna & 230V isolation). You can delete them. 
I have not found anything not compatible in their [design rules](https://aisler.community/t/pcb-design-rules/41/34)

**Comment by nliaudat** [comment]
[Gerber_PCB v54 rev 1.36 - no square holes.zip](https://github.com/nliaudat/esp32_8ch_motor_shield/files/9690015/Gerber_PCB.v54.rev.1.36.-.no.square.holes.zip) . Perhaps, you can try that with no square holes

**Reply by tango38317** [reply]
HI,
tried to upload this Gerber  files without square holes - now it seems to work without errors. i tried to go further in the assistant, but parts are missing.
i´ll have a look how to get it ready :-)
many thanks for your support.

**Comment by uplight-dev** [comment]
Hi,

I'm trying to order the PCB from this site as well.
Do you know why I get this error: "Unavailable due to missing assembly data"?
If I order without assembly, would this board be functional?

Thanks!

**Reply by nliaudat** [reply]
try directly from easyeda.com
